src/components/UserInfo/UserInfo.tsx

```diff
@@ -1,4 +1,12 @@
 // Add the required types and props
-export const UserInfo = () => (
-  <>UserInfo markup</>
+import { User } from '../../types/User';
+
+type Props = {
+  user: User;
+};
+
+export const UserInfo: React.FC<Props> = ({ user }) => (
```

Comment:
Use destructurization

```suggestion
export const UserInfo: React.FC<Props> = ({ user: { email, name } }) => (
```